doc/index.md

```diff
@@ -102,7 +102,7 @@ The ClientApp instance must be configured to match the PureCloud environment in
 
 We recommend configuring your app so the PC environment can be seeded into the url.  This allows the SDK to dynamically determine the environment at runtime and simplifies your code.
 
-1. When registering your app, configure the URL in the format: https://myapp.mydomain.com?pcEnvironment={pcEnvironment} 
+1. When registering your app, configure the URL in the format: https://myapp.mydomain.com?pcEnvironment={{pcEnvironment}}
```

Comment:
If you're saying that is supposed to be a literal value, use the HTML escape codes for open and close curlys (&#123; and &#125;). Note that HTML escape codes don't work inside code fences (single or triple backticks/tildes).

Reply:
Dug into this a bit and MD supports escaping `{` via `\{`.  Do you think this would get us past the template engine?  I can update and push if you want to test.